API.md

```diff
@@ -62,6 +62,15 @@ If no argument is provided, the method reads all channels. Otherwise, the data p
 
 If the current sensor reading does not allow understanding of the line position (some trash under the sensor or gone out of track), the previous successful estimation (which is cached) is returned. The starting value is `0.0`.
 
+### `void optimizeSensitivityOnBlackField()`
```

Comment:
It can be only if octoliner is mechanicaly or electicaly broken. In other case all 8 will "1". On realy white (not black), it will be on _sensitivity = [120-150].